docker/requirements.txt

```diff
@@ -6,3 +6,4 @@ confluent-kafka==1.7.0
 PyYAML==5.4.1
 m2crypto==0.37.1
 watchdog==2.1.2
+pyopenssl
```

Comment:
Please specify version number of the package to avoid possible future problems with newer versions.

Reply:
specified it in https://github.com/apache/nifi-minifi-cpp/pull/1219/commits/e77a2891da3ba880f1a24b8a74757d572dc2ccde